localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -623,19 +606,7 @@ void EKFLocalizer::publishEstimateResult()
   odometry.twist = twist_cov.twist;
   pub_odom_->publish(odometry);
 
-  /* debug measured pose */
```

Comment:
How about this?
- Leave `pub_measured_pose_` as it is
- Instead, remove `pub_debug_`, since its output duplicates with `pub_measured_pose_`.

From the internal conversation [here](https://star4.slack.com/archives/C4P0NSMB5/p1658391427038469?thread_ts=1658390565.200189&cid=C4P0NSMB5), it seems that at least the `pub_debug_` is no longer necessary. But I think that `pub_measured_pose_` can be used for some use case, I think we can leave it for now.

Reply:
I implemented `queue.back()` and reverted a publisher